### PR TITLE
Add a blur to backdrop images

### LINKED ISSUE #3642: Enable blurred backdrop
### Describe the feature you'd like

Option to enable blurred backdrop. When blurred background image, it creates better contrast with text, less distraction when reading and a modern look. A slow fade between backdrops when navigating is welcomed to reduce distraction.

Thank you. I love Jellyfin project.

### PR DESCRIPTION
**Pull request by haydenzeller**
**Changes**

Add a checkbox in the Theme customization view to blur the backdrop images. Add blurBackdropEnable to UserPreferences. Add _blur to BackgroundService. Add conditional modifier in AppBackground Image() to blur using androidx.compose.ui.draw.blur.

**Issues**

fix #3642

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
I don't mind adding a blur to the background, that most likely improves the design. But adding an option for it is entirely unnecessary.

**Comment by haydenzeller** [comment]
Yes I agree, another menu item is rather cumbersome. I was just unsure if you wanted this feature by default or by user pref so I added it just in case. I will update my branch.

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
Aside from the invalid formatting that needs to be fixed I found one issue, the server splash screen when authenticating should not be blurred.

File: app/src/main/java/org/jellyfin/androidtv/ui/background/AppBackground.kt [hunk 1]
```diff
@@ -62,8 +64,9 @@ private fun AppThemeBackground() {
 fun AppBackground() {
 	val backgroundService = koinInject<BackgroundService>()
 	val currentBackground by backgroundService.currentBackground.collectAsState()
+	val blurBackground by backgroundService.blurBackground.collectAsState()
 	val enabled by backgroundService.enabled.collectAsState()
-
+
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/background/AppBackground.kt:69]: fix formatting

File: app/src/main/java/org/jellyfin/androidtv/ui/background/AppBackground.kt [hunk 2]
```diff
@@ -80,7 +83,9 @@ fun AppBackground() {
 					alignment = Alignment.Center,
 					contentScale = ContentScale.Crop,
 					colorFilter = ColorFilter.tint(colorResource(R.color.background_filter), BlendMode.SrcAtop),
-					modifier = Modifier.fillMaxSize()
+					modifier = Modifier.fillMaxSize().then(
+						if (blurBackground) Modifier.blur(10.dp) else Modifier
+					)
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/background/AppBackground.kt:88]:
```suggestion
					modifier = Modifier
						.fillMaxSize()
						.then(if (blurBackground) Modifier.blur(10.dp) else Modifier)
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

**Review by nielsvanvelzen — APPROVED** [review]
Awesome!

**Comment by acordtag** [comment]
Hello, where does the 'enable' option appear within the Android TV interface? - cannot be seen on the 'Customization' section; it only appear the "Show backdrops" option only.

Currently running Server 10.9.9 and AndroidTV Jellyfin

**Comment by nielsvanvelzen** [comment]
Background blur is always enabled on supported devices (Android 12 and newer).

**Comment by Antonio1637** [comment]
Hello. Is it possible to disable this features. I have a headache while reading text and scrolling. Blur effect is across whole screen and it does not like nice.

**Comment by nielsvanvelzen** [comment]
Please open an issue for that. If there's enough demand we may add an option for it.

**Comment by shahribms** [comment]
@nielsvanvelzen this should count as demand enough. referencing both issues https://github.com/jellyfin/jellyfin-androidtv/issues/4716

**Pull request merged**